include/common/errors.h

```diff
@@ -211,12 +211,27 @@
 
 #define SCE_ERROR_KERNEL_ERROR                                  0x80020001
 #define SCE_ERROR_KERNEL_NOT_IMPLEMENTED                        0x80020002
+#define SCE_ERROR_KERNEL_SYSCALLTABLE_ALREADY_REGISTERED        0x80020036

```

Comment:
Not a requested change as of now, only a question: Do we want to use the official Sony error names where available? In this case, the error code is originally defined as this:
```c++
#define SCE_KERNEL_ERROR_SYCALLTABLE_USED 0x80020036
```

Granted, I always felt like some of these namings to be of rather poor quality and some of the alternative names (i.e. provided by the PSP SDK) to better convey the actual error state they represent.

Futhermore...we aren't consistently using Sony-only error names anyway currently (which is a relict of us building this file originally from open-source content (PSPSDK) and only later starting to use the Sony used names):
```c++
#define SCE_KERNEL_ERROR_ILLEGAL_CONTEXT 0x80020064
```
and the name we use:
```c++
#define SCE_ERROR_KERNEL_CANNOT_BE_CALLED_FROM_INTERRUPT        0x80020064
```

I think we can keep using error codes with improved naming over the original names just fine.

Reply:
I guess if Sony's name is publicly known we could define both, or put a comment about the original name. I don't think it matters so much and I'm fine with it so please merge it if you're fine with it too. :)

Reply:
I don't think we need to define both (comment perhaps, but probably also not really required). I will just file us using improved custom naming under "creative freedom".